src/app/routes/enrolment/view-requests/components/expiration-date/expiration-date.component.spec.ts

```diff
@@ -99,7 +99,7 @@ describe('ExpirationDateComponent', () => {
 
     const { expirationDateInput } = getSelectors(hostDebug);
 
-    expect(new Date(expirationDateInput.value)).toEqual(new Date(2000, 0, 2));
+    expect(new Date(expirationDateInput.value)).toEqual(new Date(2000, 0, 1));
```

Comment:
Is this test checking right now something? I'm not sure, because default date is 1st January 2000, and after setting defaultValidityPeriod it is still 1st January 2000, but with different hour/minute/seconds. To adjust test change defaultValidityPeriod from 1 day in seconds into 1 day in miliseconds.
```
    component.defaultValidityPeriod = 60 * 60 * 24 * 1000;
```
This way we will know that setting defaultValidityPeriod is changing input.